rpc/lib/doc.go

```diff
@@ -1,3 +1,5 @@
+package rpc
```

Comment:
@melekes could we instead move this to the bottom and no line after the last `*/`? e.g
```go
/*
Docs
*/
package rpc
```

With the current change, it won't show up in the overview. Nice working putting it in the proper package!

Reply:
done